### ISSUE TITLE
EXEC_PARAMS in #!python3 mode

### ISSUE THREAD

**Issue by max-bold** [eirannejad/pyRevit#1359]
Good time of the day!

There are two issues in one place:

1. In #!python3 mode importing `EXEC_PARAMS` from `pyrevit` throws an exception `"AttributeError : module 'clr' has no attribute 'AddReferenceToFileAndPath'"`
2. More strange one. To overcome the first issue in #!python3 mode , I've tried to use old method `__shiftclick__`. But no luck. I can read its value with `print(__shiftclick__)`, but when using it in `if` clause it always falls in else. The sample code is below:

```
#!python3
print(__shiftclick__)
if __shiftclick__:  
    print('__shiftclick__')
else:
    print('not __shiftclick__')
```

This is pyrevit env output:

==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.8.8" | Path: "D:\PyRevit"
==> Attachments
master | Product: "2021 First Customer Ship" | Engine: IPY277 (277) | Path: "D:\PyRevit"
master | Product: "2019.2 (Update)" | Engine: IPY277 (277) | Path: "D:\PyRevit"
==> Installed Extensions
LAB | Type: Unknown | Repo: "" | Installed: "C:\Users\user\Desktop\ForRevit\pyRevitScrips\LAB.extension"
==> Default Extension Search Path
C:\Users\user\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
C:\Users\user\Desktop\ForRevit\pyRevitScrips
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
2021 First Customer Ship | Version: 21.0.0.383 | Build: 20200220_1100(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2021\"
2019.2 (Update) | Version: 19.2.0.65 | Build: 20181217_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019\"
==> Running Revit Instances
PID: 4896 | 2021 First Customer Ship | Version: 21.0.0.383 | Build: 20200220_1100(x64) | Language: 0 | Path: "C:\Program Files\Autodesk\Revit 2021"
==> User Environment
Microsoft Windows 10 [Version 10.0.19042]
Executing User: BOLDOREV\user
Active User:
Admin Access: No
%APPDATA%: "C:\Users\user\AppData\Roaming"
Latest Installed .Net Framework: 4.8
Installed .Net Target Packs: v3.5 v4.0 v4.5 v4.5.1 v4.5.2 v4.6 v4.6.1 v4.7.2 v4.X
Installed .Net-Core Target Packs: v3.1.414 v5.0.301 v5.0.303
pyRevit CLI 4.8.8.0

**Comment by alexdaversa** [comment]
@eirannejad, this seems like the root cause of #1288 #1311 as well. Tillman in #1288 gave this stack overflow response to replace AddReferenceToFileAndPath with AddReference. https://stackoverflow.com/questions/49998309/import-of-dll-with-pythonnet

Resolving this error seems like a good next step in providing python3 compatibility.

**Comment by eirannejad** [comment]
Pythonnet is in the midst of a huge improvement (pythonnet v3) and I'm kinda waiting for that to be released so I can merge into pyRevit. Once that step is done I'll start working on getting the `pyrevit` module compatible with cpython. Closing this for now.

Continue conversation under https://github.com/eirannejad/pyRevit/issues/1438
